src/sentry/issues/auto_source_code_config/frame_info.py

```diff
@@ -20,14 +23,53 @@
 UNSUPPORTED_NORMALIZED_PATH_PATTERN = re.compile(r"^[^/]*$")
 
 
-class FrameInfo:
-    def __init__(self, frame: Mapping[str, Any], platform: str | None = None) -> None:
-        if platform:
-            platform_config = PlatformConfig(platform)
-            if platform_config.extracts_filename_from_module():
```

Comment:
This removed block is what would make the frame info be determined based on the filename and module.

`frame_info_from_module` will now only belong to `ModuleBasedFrameInfo`.

![Image](https://github.com/user-attachments/assets/91a22f39-9153-407e-b7c4-42f901b17af5)